#!/usr/bin/python3
def raise_exception():
    try:
        i = 5 + ""
    except:
        raise
